src/platform/silabs/wifi/wf200/WifiInterface.cpp

```diff
@@ -436,7 +444,7 @@ static void sl_wfx_scan_result_callback(sl_wfx_scan_result_ind_body_t * scan_res
     struct scan_result_holder * ap;
 
     ChipLogDetail(DeviceLayer, "# %2d %2d  %03d %02X:%02X:%02X:%02X:%02X:%02X  %s", scan_count, scan_result->channel,
-                  ((int16_t) (scan_result->rcpi - 220) / 2), scan_result->mac[0], scan_result->mac[1], scan_result->mac[2],
+                  ((int16_t) (ConvertRcpiToRssi(scan_result->rcpi))), scan_result->mac[0], scan_result->mac[1], scan_result->mac[2],
```

Comment:
Is there int16_t still necessary here since the function returns a int16_t?

Reply:
removed.